### ISSUE TITLE
Unilocular cyst

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10114]
**Preferred term label:**
Unilocular cyst

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1265786
https://www.ncbi.nlm.nih.gov/medgen/C1336868

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Ovarian cystadenoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
should this be a modifier (Unilocular) or a specific term for a type of ovarian cyst? The latter would only be appropriate if there is an important clinical distinction between uni and multilocular (and then we would need both terms).

**Comment by MickeySegal** [comment]
The name we use is "Modifier: Cyst is unilocular on ultrasound".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000631
- label: Unilocular ovarian cyst
- parents: [Ovarian cyst (HP:0000138)](https://hpo.jax.org/app/browse/term/HP:0000138)
- pmid: PMID:25797108
- def: An ovarian cyst without septa and a solid component.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10114
- synonyms:
